datatables.go

```diff
@@ -285,11 +285,14 @@ func (p *Parser) bindGrenadeProjectiles(entity *st.Entity) {
 	})
 
 	entity.OnDestroy(func() {
-		p.eventDispatcher.Dispatch(events.NadeProjectileDestroyedEvent{
-			Projectile: proj,
-		})
+		// Issue #42
+		// Sometimes entities are destoyed even after round_officially_ended, sometimes not
+		// Due to that we need to check if we already deleted it, otherwise we get a nil pointer dereference
```

Comment:
Consider re-reading this comment :slightly_smiling_face:

Reply:
Sorry, I'm not quite following you here? Could you elaborate?

Reply:
Maybe it's just me, but I find that it's too general for what it actually covers.

Maybe something like the following?
> Sometimes grenade projectile entities are automatically destroyed after round_officially_ended, and sometimes they are not.
> The below check accommodates for the times they are not

Reply:
> The below check accommodates for the times they are not.
Actually the opposite! :slightly_smiling_face: but I've changed it to hopefully be more clear.